utils/webassembly/distribute-latest-toolchain.sh

```diff
@@ -103,22 +103,22 @@ unzip ubuntu18.04-installable.zip
 unzip ubuntu20.04-installable.zip
 unzip macos-installable.zip
 
-toolchain_name=$(basename $(tar tfz swift-wasm-$2-SNAPSHOT-ubuntu18.04.tar.gz | head -n1))
+toolchain_name=$(basename $(tar tfz swift-wasm-$2-SNAPSHOT-ubuntu18.04-x86_64.tar.gz | head -n1))
 
 if is_released $toolchain_name; then
   echo "Latest toolchain $toolchain_name has been already released"
   exit 0
 fi
 
-mv swift-wasm-$2-SNAPSHOT-ubuntu18.04.tar.gz "$toolchain_name-ubuntu18.04.tar.gz"
-mv swift-wasm-$2-SNAPSHOT-ubuntu20.04.tar.gz "$toolchain_name-ubuntu20.04.tar.gz"
-mv swift-wasm-$2-SNAPSHOT-osx.tar.gz "$toolchain_name-osx.tar.gz"
+mv swift-wasm-$2-SNAPSHOT-ubuntu18.04-x86_64.tar.gz "$toolchain_name-ubuntu18.04-x86_64.tar.gz"
+mv swift-wasm-$2-SNAPSHOT-ubuntu20.04-x86_64.tar.gz "$toolchain_name-ubuntu20.04-x86_64.tar.gz"
+mv swift-wasm-$2-SNAPSHOT-macos-x86_64.tar.gz "$toolchain_name-macos-x86_64.tar.gz"
 
 create_tag $toolchain_name $head_sha
 release_id=$(create_release $toolchain_name $toolchain_name $head_sha)
 
-upload_tarball $release_id "$toolchain_name-ubuntu18.04.tar.gz"
-upload_tarball $release_id "$toolchain_name-ubuntu20.04.tar.gz"
-upload_tarball $release_id "$toolchain_name-osx.tar.gz"
+upload_tarball $release_id "$toolchain_name-ubuntu18.04-x86_64.tar.gz"
+upload_tarball $release_id "$toolchain_name-ubuntu20.04-x86_64.tar.gz"
+upload_tarball $release_id "$toolchain_name-macos-x86_64-x86_64.tar.gz"
```

Comment:
@MaxDesiatov Sorry, I overlooked this. https://github.com/swiftwasm/swift/pull/1919#discussion_r499640381